### PR TITLE
Tweak event aggregations exposed on /metrics

### PR DESCRIPTION
**Pull request by frederikrothenberger**
This PR changes the aggregations exposed on /metrics to be more useful:
* remove the metrics for the non-official / other domain
* add prepare delegation count for https://identity.ic0.app

This allow to have both, aggregated session time _and_ count be displayed on the Grafana dashboard, while keeping the number of metrics tracked by victoria metrics low.

<hr/><details><summary>🟡 Some screens were changed</summary></details>

### CODE REVIEW

File: src/internet_identity/src/http/metrics.rs [hunk 1]
```diff
@@ -152,6 +152,11 @@ fn event_metrics(w: &mut MetricsEncoder<Vec<u8>>) -> Result<(), std::io::Error>
         "Cumulative authenticated session duration in seconds for the given front-end origin.",
     )?;
     pd_aggregation_metrics(PD_SESS_SEC, sess_sec_builder)?;
+    let count_builder = w.gauge_vec(
```
Comment (lmuntaner) [inline src/internet_identity/src/http/metrics.rs:155]: Why wasn't this before exposed in /metrics ?

File: src/internet_identity/src/http/metrics.rs [hunk 2]
```diff
@@ -160,19 +165,17 @@ fn pd_aggregation_metrics(
     mut metrics_builder: LabeledMetricsBuilder<Vec<u8>>,
 ) -> Result<(), std::io::Error> {
     for window in &[Day, Month] {
-        for domain in &[None, Some(Ic0App), Some(InternetComputerOrg)] {
```
Comment (lmuntaner) [inline src/internet_identity/src/http/metrics.rs:163]: Is there a specific reason to remove the metrics from the non-official domain?

File: src/internet_identity/tests/integration/http.rs [hunk 3]
```diff
@@ -609,9 +609,9 @@ fn should_list_aggregated_session_seconds_and_event_data_counters() -> Result<()
     api::prepare_delegation(
         &env,
         canister_id,
-        test_authn_method().principal(),
+        authn_method_ic0.principal(),
```
Comment (lmuntaner) [inline src/internet_identity/tests/integration/http.rs:612]: Why this change?

File: src/internet_identity/tests/integration/http.rs [hunk 4]
```diff
@@ -609,9 +609,9 @@ fn should_list_aggregated_session_seconds_and_event_data_counters() -> Result<()
...
         user_number_1,
-        "https://some-dapp-1.com",
+        "https://some-dapp-2.com",
```
Comment (lmuntaner) [inline src/internet_identity/tests/integration/http.rs:614]: Why this change?

File: src/internet_identity/tests/integration/http.rs [hunk 5]
```diff
@@ -637,33 +637,23 @@ fn should_list_aggregated_session_seconds_and_event_data_counters() -> Result<()
...
     assert_metric(
         &metrics,
-        "internet_identity_prepare_delegation_session_seconds{dapp=\"https://some-dapp-1.com\",window=\"24h\",ii_origin=\"other\"}",
-        5400f64,
-    );
-    assert_metric(
-        &metrics,
-        "internet_identity_prepare_delegation_session_seconds{dapp=\"https://some-dapp-1.com\",window=\"30d\",ii_origin=\"other\"}",
+        "internet_identity_prepare_delegation_session_seconds{dapp=\"https://some-dapp-2.com\",window=\"24h\",ii_origin=\"ic0.app\"}",
         5400f64,
```
Comment (lmuntaner) [inline src/internet_identity/tests/integration/http.rs:641]: Sorry, but why is it 5400?

I see that the session lasts 3600 one time and None the second.

File: src/internet_identity/src/http/metrics.rs (hunk 1 above)
Comment (frederikrothenberger) [reply src/internet_identity/src/http/metrics.rs:155]: Because there is non-negligible cost to the time-series kept in victoria metrics. So we only expose a specific subset to be available for historical data.

And this PR changes the subset to be more useful.

File: src/internet_identity/src/http/metrics.rs (hunk 2 above)
Comment (frederikrothenberger) [reply src/internet_identity/src/http/metrics.rs:163]: Yes, because of the non-negligible storage cost and because the data is not that interesting.

File: src/internet_identity/tests/integration/http.rs (hunk 3 above)
Comment (frederikrothenberger) [reply src/internet_identity/tests/integration/http.rs:612]: So that the counter of `"https://some-dapp-2.com"` is not just 1, but a higher number so that it couldn't just be an off by one error 😉

File: src/internet_identity/tests/integration/http.rs (hunk 4 above)
Comment (frederikrothenberger) [reply src/internet_identity/tests/integration/http.rs:614]: see above

File: src/internet_identity/tests/integration/http.rs (hunk 5 above)
Comment (frederikrothenberger) [reply src/internet_identity/tests/integration/http.rs:641]: Because `None` defaults to 1800 seconds (30 minutes). I can also set an explicit value if you prefer.

File: src/internet_identity/src/http/metrics.rs (hunk 1 above)
Comment (lmuntaner) [reply src/internet_identity/src/http/metrics.rs:155]: Ah, ok, understood.

File: src/internet_identity/src/http/metrics.rs (hunk 2 above)
Comment (lmuntaner) [reply src/internet_identity/src/http/metrics.rs:163]: Thanks for the explanation!

File: src/internet_identity/tests/integration/http.rs (hunk 3 above)
Comment (lmuntaner) [reply src/internet_identity/tests/integration/http.rs:612]: Ok thanks for the explanation.

File: src/internet_identity/tests/integration/http.rs (hunk 5 above)
Comment (lmuntaner) [reply src/internet_identity/tests/integration/http.rs:641]: Ah, ok, got it.

**Review by lmuntaner — APPROVED** [review]
